### ISSUE TITLE
Data Futures Lab landscape analysis: Create a home (follow up ticket)

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/foundation.mozilla.org#4915]
**Project expected soft launch:** Sept 2nd

Follow up on #4830 - MOOD: Not too much fun but not too scientific and boring. 

Home was created  - [here is the link](https://foundation.mozilla.org/en/initiatives/data-futures/) - based on [latest copy](https://docs.google.com/document/d/1X1kAvb0-e7zzYS-eXFiTns6lXQmstxVI7GtWuH3BhPA/edit) (received on Aug 11)
URL was defined: `/initiatives/data-futures/`

### Deliverables - EDITED on Aug 27th:
- [x] Build new version on production site - based on [latest copy](https://docs.google.com/document/d/1X1kAvb0-e7zzYS-eXFiTns6lXQmstxVI7GtWuH3BhPA/edit) (received on Aug 11)
- [x] Asked Eeva to place the new copy in the CMS 
- [x] Remove “lab” from data futures URL
- [x] Password-protect the site - Taís will create a password and send to the team.
- [ ] Think about where Data Futures will show on the site: send suggestions to the team - this can happen AFTER deadline, we need some planning.
- [x] Airtable: link to **projects** tab - embed is more like a teaser
- [x] Remove the sign up email list for now
- [x] Try narrow and full width options
- [x] add more colors/make it more beautiful
- [x] Illustration: 4 illustrations - top (header), overview and for each one of the parts of the report (3 parts) - add a mini header to every section, color coded as the PDF report. See (color scheme of airtable into the site).
- [ ] Color of the numbers: change mood: add shades, more dynamic, away from academia, a little fun.
- [x] Data for Empowerment (use About instead of Overview because we already have 1 Overview we can't change)
- [ ] Download Report section: 2 PDFs (?) - 1 overview and 1 for each tab...maybe have an illustration of a physical report to encourage PDF (example: stakeholder research mockup). Different subjects. They could be volumes (vol 1, 2, 3) or parts (I, II, III)
- [ ] do a design pass once it is done
- [ ] Create illustration/photographic style graphic social media 
- [x] Edit [kumu image](https://kumu.io/JvanGeuns/data-stewardship-mapping-64b2): 
- Remove all text in grey (4 sentences about “rights give raise to..”)
- Change “ Digital fiduciaries” to “Data fiduciaries”
- Export to JPEG and send to them

~- [ ] Add newsletter signup (can put IHR snippet for now but final signup is blocked by #4848)~

**Comment by taisdesouzalessa** [comment]
Pausing this project during this sprint (July 21-Aug03): waiting for the Insights team to move on this project a bit more before we get into the design stage. We will connect with that team next sprint and reassess if it is ready for design.

**Comment by taisdesouzalessa** [comment]
Original ticket was updated based on latest conversation with Eeva.

**Comment by taisdesouzalessa** [comment]
@mehan - tagging you here so you are aware of the project's progress.

**Comment by taisdesouzalessa** [comment]
Started adding some content to "Data for Empowerment" tab to see how we can use existing Wagtail Features to Breakdown huge pieces of content into digestible blocks. https://foundation.mozilla.org/en/initiatives/data-futures-lab/database/

**Comment by taisdesouzalessa** [comment]
Blocked by getting more info from Insights team. We will have a meeting on **Aug 27th (Thursday)** that may unblock this work.

**Comment by taisdesouzalessa** [comment]
Notes from the [meeting here](https://docs.google.com/document/d/1z6hC6o6accxz6rtHciTHPnlO6GEVlIyppjxwEVdwI44/edit?usp=sharing)

To do by Tuesday: Sept 1st
- [ ] items from the list - except illustrations if there isn't time.

**Comment by taisdesouzalessa** [comment]
Worked on the heavy lift items of this page today:

- headers for all sections 
- image for intro page
- color coded the 3 tabs: “data for empowerment”/”who is trying”/”what could go wrong”
- did a design review of “overview page” content

Site here (Password protected, password is in the doc linked below): https://foundation.mozilla.org/en/initiatives/data-futures/

[Left notes on the doc](https://docs.google.com/document/d/1z6hC6o6accxz6rtHciTHPnlO6GEVlIyppjxwEVdwI44/edit?usp=sharing) for the Insights team to review.

**Comment by taisdesouzalessa** [comment]
Deadline changed to EOW. Added a new ticket for current sprint here: #5163 - closing this one.
